src/compas_model/elements/element.py

```diff
@@ -5,723 +5,402 @@
-                        "is_visible": True,
-                    },
-                ),
-                (
-                    "geometry",
-                    {
-                        "facecolor": [0.9, 0.9, 0.9],
-                        "linecolor": [0.0, 0.0, 0.0],
-                        "linewidth": 1,
-                        "opacity": 0.75,
-                        "edges": True,
-                        "is_visible": True,
-                    },
-                ),
-                (
-                    "frame",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 1.0,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "aabb_mesh",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "oobb_mesh",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "convex_hull",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-            ]
+        return (
+            [g.copy() if g and copy_flag else g for g in geometries]
+            if isinstance(geometries, list)
+            else [geometries.copy()]
+            if copy_flag
+            else [geometries]
         )
 
-        return ordered_dict
-
     # ==========================================================================
     # Serialization
     # ==========================================================================
 
     @property
     def data(self):
-        # create the data object from the class properties
-        # call the inherited Data constructor for json serialization
+
         data = {
             "name": self.name,
-            "id": self.id,
             "frame": self.frame,
             "geometry_simplified": self.geometry_simplified,
             "geometry": self.geometry,
+            "copy_geometry": self.copy_geometry,
```

Comment:
?

Reply:
Added explanation to the docstring:
```python
    copy_geometry : bool, optional
        If True, the geometry will be copied to avoid references.
```